bpf/gotracer/go_nethttp.c

```diff
@@ -558,6 +561,16 @@ int beyla_uprobe_roundTripReturn(struct pt_regs *ctx) {
     bpf_dbg_printk(
         "status %d, offset %d, resp_ptr %lx", trace->status, status_code_ptr_pos, (u64)resp_ptr);
 
+    u64 response_length_ptr_pos = go_offset_of(ot, (go_offset){.v = _response_length_ptr_pos});
+    bpf_probe_read(&trace->response_length,
+                   sizeof(trace->response_length),
+                   (void *)(resp_ptr + response_length_ptr_pos));
+
+    bpf_dbg_printk("response_length %d, offset %d, resp_ptr %lx",
```

Comment:
```suggestion
    bpf_dbg_printk("response_length %llu, offset %llu, resp_ptr %llx",
```
(it's possible other parts of our code carry the same mistake, so I understand if you were just being consistent here)

Reply:
Thank you, corrected